plugins/inputs/procstat/procstat.go

```diff
@@ -324,7 +324,7 @@ func (p *Procstat) gatherOld(acc telegraf.Accumulator) error {
 
 func (p *Procstat) gatherNew(acc telegraf.Accumulator) error {
 	now := time.Now()
-
+	running := make(map[PID]bool)
```

Comment:
had to move this outside the filter loop as later filters were clearing processes from earlier filters.